blog/releases/announcing-knative-v0-22-release.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Version v0.22 release"
+linkTitle: "Version v0.22 release"
+Author: "[Carlos Santana](https://twitter.com/csantanapr)"
+Author handle: https://github.com/csantanapr
+date: 2021-04-06
+description: "Knative v0.22 release announcement"
+type: "blog"
+image: knative-eventing.png
+---
+
+
+### Announcing Knative v0.22 Release
+
+A new version of Knative is now available across multiple components.
+Follow the instructions in the documentation [Installing Knative](https://knative.dev/docs/install/) for the respective component.
+
+#### Table of Contents
+- [Serving v0.22](#serving-v022)
+- [Eventing v0.22](#eventing-v022)
+- Eventing Extensions
+    - [Eventing Kafka Broker v0.22](#eventing-kafka-broker-v022)
+- `kn` [CLI v0.22](#client-v022)
+    - `kn` [CLI Plugins](#cli-plugins)
+- [Operator v0.22](#operator-v022)
+- [Thank you contributors](#thank-you-contributors)
+
+
+
+### Highlights
+
+- Serving Domain Mapping improves multi-tenency support to avoid domain name 'squatting' in cluster.
+- Eventing now allows Trigger users to set another namespace's Subscriber
```

Comment:
```suggestion
- Eventing now allows subscribers and triggers from different namespaces to be used together.
```

Not totally sure if this is right or what this means otherwise, it seems like you're saying triggers can have subscribers that are in a different namespace to them now?

Reply:
yes, I think so